pyiron/atomistics/structure/atoms.py

```diff
@@ -1227,30 +1227,102 @@ def _get_flattened_orientation(view_plane, distance_from_camera):
-        if distance_from_camera <= 0:
-            raise ValueError("´distance_from_camera´ must be a positive float!")
         view_plane = np.array(view_plane).reshape(-1, 3)
         rotation_matrix = np.roll(np.eye(3), -1, axis=0)
         rotation_matrix[:len(view_plane)] = view_plane
         rotation_matrix /= np.linalg.norm(rotation_matrix, axis=-1)[:, np.newaxis]
         rotation_matrix[1] -= np.dot(rotation_matrix[0], rotation_matrix[1]) * rotation_matrix[0]  # Gran-Schmidt
         rotation_matrix[2] = np.cross(rotation_matrix[0], rotation_matrix[1])  # Specify third axis
         if np.isclose(np.linalg.det(rotation_matrix), 0):
-            rotation_matrix = np.eye(3)  # view_plane = [0,0,1] is the default view of NGLview, so we do not modify it
-        else:
-            rotation_matrix = np.roll(rotation_matrix / np.linalg.norm(rotation_matrix, axis=-1)[:, np.newaxis], 2, axis=0).T
+            return np.eye(3)  # view_plane = [0,0,1] is the default view of NGLview, so we do not modify it
+        return np.roll(rotation_matrix / np.linalg.norm(rotation_matrix, axis=-1)[:, np.newaxis], 2, axis=0).T
+
+    def _get_flattened_orientation(self, view_plane, distance_from_camera):
+        """
+        A helper method to plot3d, which generates a rotation matrix from the input `view_plane`, and returns a
+        flattened list of len = 16. This flattened list becomes the input argument to `view.contol.orient`.
+
+        Args:
+            view_plane (numpy.ndarray/list): A Nx3-array/list (N = 1,2,3); the first 3d-component of the array
+                specifies which plane of the system to view (for example, [1, 0, 0], [1, 1, 0] or the [1, 1, 1] planes),
+                the second 3d-component (if specified, otherwise [1, 0, 0]) gives the horizontal direction, and the
+                third component (if specified) is the vertical component, which is ignored and calculated internally.
+                The orthonormality of the orientation is internally ensured, and therefore is not required in the
+                function call.
+            distance_from_camera (float): Distance of the camera from the structure. Higher = farther away.
+
+        Returns:
+            (list): Flattened list of len = 16, which is the input argument to `view.contol.orient`
+        """
+        if distance_from_camera <= 0:
+            raise ValueError("´distance_from_camera´ must be a positive float!")
         flattened_orientation = np.eye(4)
-        flattened_orientation[:3, :3] = rotation_matrix
+        flattened_orientation[:3, :3] = self._get_orientation(view_plane)
 
         return (distance_from_camera * flattened_orientation).ravel().tolist()
 
+    def plot3d_plotly(
+        self,
+        scalar_field=None,
+        particle_size=1.0,
+        camera="orthographic",
+        view_plane=np.array([1, 1, 1]),
+        distance_from_camera=1.25,
+    ):
+        """
+
```

Comment:
Docstring before args needs some short description like "Make a 3D plot of the atomic structure."